cocotb/share/makefiles/Makefile.sim

```diff
@@ -56,7 +56,7 @@ endif
 # recompilation of the simulation if cocotb is updated.
 CUSTOM_SIM_DEPS += $(shell find $(COCOTB_PY_DIR)/cocotb/ -name "*.py")
 
-include $(COCOTB_SHARE_DIR)/lib/Makefile
+#include $(COCOTB_SHARE_DIR)/lib/Makefile
```

Comment:
So actually, this whole file could be removed?: https://github.com/cocotb/cocotb/blob/master/cocotb/share/lib/Makefile

Reply:
yes and many more

Reply:
Can we stick a to-do comment at the top of this makefile that it isn't used any more?

Reply:
All makefiles in `cocotb/share/lib` can/should be removed.